src/datastore/pg-store-v2.ts

```diff
@@ -518,6 +518,12 @@ export class PgStoreV2 extends BasePgStoreModule {
             FROM nft_events
             WHERE sender = ${args.address} OR recipient = ${args.address}
           )
+        ),
+        total_count AS (
+          SELECT COUNT(*)::int AS total_count
+          FROM address_txs
+          INNER JOIN txs USING (tx_id, index_block_hash, microblock_hash)
+          WHERE canonical = TRUE AND microblock_canonical = TRUE
```

Comment:
Instead of this join, add the `WHERE canonical = TRUE AND microblock_canonical = TRUE` filters to the union tables above